graphdatascience/graph/graph_proc_runner.py

```diff
@@ -43,6 +44,7 @@ def _path(package: str, resource: str) -> ContextManager[pathlib.Path]:
 
     @client_only_endpoint("gds.graph")
     def load_cora(self, graph_name: str = "cora", undirected: bool = False) -> Graph:
```

Comment:
here we dont have doc string to document the args?
Is the idea to add it later?